css/style.css

```diff
@@ -0,0 +1,1973 @@
+  padding-right: 40px;
+}
+
+.testDrive__image-ellipse {
+  position: absolute;
+  width: 22px;
+  height: 22px;
+  top: -11px;
+  left: 10px;
+}
+
+.testDrive__image-tesla {
+  position: absolute;
+  width: 14.18px;
+  height: 8.23px;
+  left: 14px;
+  top: -1px;
+  animation-duration: 3s;
+  animation-name: rotate;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  transform-origin: 50% 0%;
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(-230deg);
+  }
+}
+
+.testDrive__image-dott {
+  position: absolute;
+  width: 2.58px;
+  height: 2.58px;
+  left: 20px;
+  top: -3px;
+}
+
+.testDrive__title {
+  margin: 0px;
+  padding-left: 0px;
+  font-size: 13px;
+  font-family: "MontserratBold";
+}
+
+.description-characteristic__button {
+  margin: 0px;
+  color: #E51937;
+  display: flex;
+  align-self: center;
+  font-family: "MontserratBold";
+  font-size: 13px;
+  border: 2px solid #E51937;
+  box-shadow: none;
+  transition: box-shadow 0.6s linear;
+  padding: 10px 40px;
+}
+
+.description-characteristic__button:hover {
+  box-shadow: 0px 0px 0px 2px #E51937;
+}
+
+.description-characteristic__testDrive-phone {
+  display: none;
+}
+
+.description-characteristic__button-phone {
+  display: none;
+}
+
+.characteristic__arrowDown {
+  display: flex;
+  justify-content: center;
+  padding-top: 37px;
+  position: relative;
+  padding-left: 70px;
+  animation: arrow 1s linear infinite;
+}
+
+@keyframes arrow {
+  0% {
+      bottom:0;
+  }
+  50% {
+      bottom:10px;
+  }
+  100% {
+      bottom:0;
+  }
+}
+
+.arrowDown__icon {
+  position: relative;
+  border: 3px solid  #E51937;
+  border-width: 0 0 2px 2px;
+  width: 16px;
+  height: 16px;
+  line-height: 0;
+  font-size: 0;
+  -ms-transform: rotate(-135deg);
+  -o-transform: rotate(-135deg);
+  transform: rotate(-45deg);
+  top: 5px;
+  cursor: pointer;
+}
+
+.inside {
+  padding-top: 37px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
+}
+
+.inside__title {
+  font-size: 70px;
+  font-family: "MontserratSemiBold";
+  padding: 64px 80px;
+  margin: 0;
+  color: #E51937;
+}
+
+.inside__image {
+  position: absolute;
+  z-index: -1;
+  bottom: 0;
+  height: 85%;
+}
+
+.inside__description {
+  background: #E51937;
+  color: white;
+  width: 26%;
+  padding: 51px 44px 63px 52px;
+}
+
+.description-inside__title {
+  font-family: "MontserratSemiBold";
+  font-size: 30px;
+  width: 100%;
+  text-align: justify;
+  margin: 0;
+}
+
+.description-inside__text {
+  font-family: "MontserratMedium";
+  font-size: 15px;
+  width: 97%;
+  padding-bottom: 50px;
+}
+
+.description-inside__display {
+  display: flex;
+  flex-direction: row;
+}
+
+.display-inside__icon {
+  position: relative;
+}
+
+.display-inside__image {
+  position: relative;
+  z-index: 0;
+}
+
+.display-inside__icon-wrapper {
+  position: absolute;
+  z-index: 1;
+  top: 33px;
+  left: 13px;
+} 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5520132)